src/package.json

```diff
@@ -3,14 +3,19 @@
   "version": "1.0.0",
   "description": "Monitor V2",
   "type": "module",
+  "engines": {
+    "node": "18.12.x",
+    "npm": "8.x"
+  },
   "scripts": {
     "prestart": "npm run build",
     "start": "node app.js",
     "dev": "nodemon app.js",
-    "build": "node esbuild & npm run build:png && npm run build:svg && npm run copy:root",
-    "build:png": "squoosh-cli --oxipng '{level: 2}' --quant '{numColors: 255}' -d ../dist/images ./client/images/*.png",
+    "build": "node esbuild & npm run copy:root & npm run copy:webp & npm run build:svg",
     "build:svg": "svgo -f ./client/images/ -o ../dist/images",
+    "copy:webp": "rsync -d --include='*.webp' ./client/images/ ../dist/images/",
```

Comment:
is there a reason that `copy:webp` and `copy:root` cannot be combined into one command?

Reply:
Since we shouldn't have anything other than `.webp` in `./client/images/`, can we just remove `copy:webp`?

Reply:
we have and will continue to have quite a few .svg in there as well...

Reply:
Ah gotcha thanks

Reply:
Actually – I was considering adding a script in the future to handle .svg the same way as .webp to keep them consistent.  i.e. compress .svg locally, manually via script.  When that's done, we could copy the entire image folder as you suggest...